201-logic-app-veter-pipeline/azuredeploy.parameters.json

```diff
@@ -0,0 +1,6 @@
+﻿{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
@gatneil Since the function has to exist already, I am wondering what would be the best practice for such template parameters. In current experience, user needs to create the function app container first and then copy the same name in template parameter here.
I am going to add the link to Create an Azure Functions container - https://ms.portal.azure.com/#create/Microsoft.FunctionApp in the readme file. So that should help with the creation of function app container step for users.